### DISCUSSION TITLE
Permanent 500 Server Error on a particular issue

### DISCUSSION THREAD

**Discussion by rasagy** [community/community#65269] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

A specific issue on my course repo seems to only show 500 server error for over two days: https://github.com/mapschool/course/issues/77

I noticed an [earlier discussion](https://github.com/orgs/community/discussions/23940) on exactly this, which was flagged as a 🐛 and promptly fixed by @canuckjacq, but I’m not sure if there’s something I can do to fix this on my side.

Haven’t found anything on Google/FAQ/docs that could help. Feel free to point me to a link in case there’s a quick DIY fix.

Thanks!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by nirav-gajera** [comment]
you should contact [Github Support](https://help.github.com/)

**Comment by rasagy** [comment]
Thanks. Didn’t get a response from there, but it seems to be working fine after 2 days. Closing, will share if I get any more context on what was the cause.
